README.md

```diff
@@ -104,201 +107,4 @@ aws lambda invoke \
-  - check samplesheet_check.py exists
-2. Call the run_sample_sheet_content_check function
-3. Call the run_sample_sheet_check_with_metadata function
-'
-
-# Set to fail
-set -euo pipefail
-
-### USER ####
-
-SAMPLESHEET_FILE="SampleSheet.csv" 
-
-#############
-
-
-## GLOBALS ##
-
-SAMPLESHEET_CHECK_SCRIPT="lambdas/functions/samplesheet_check.py"
-CONDA_ENV_NAME="samplesheet-check-backend"
-
-#############
-
-
-### SETUP ###
-
-if [[ -z "${PORTAL_TOKEN-}" ]]; then
-  echo "Error: Could not get the env var 'PORTAL_TOKEN'. Exiting" 1>&2
-  exit 1
-fi
-
-if [[ -z "${data_portal_domain_name-}" ]]; then
-  echo "Error: Could not get the env var 'data_portal_domain_name'. Exiting" 1>&2
-  exit 1
-fi
-
-if [[ ! -f "${SAMPLESHEET_FILE}" ]]; then
-  echo "Error: Could not find the file '${SAMPLESHEET_FILE}'" 1>&2
-  exit 1
-fi
-
-if [[ ! -f "${SAMPLESHEET_CHECK_SCRIPT}" ]]; then
-  echo "Error: Could not find the file '${SAMPLESHEET_CHECK_SCRIPT}'" 1>&2
-  exit 1
-fi
-
-#############
-
-
-### TESTS ###
-
-python_file="$(mktemp)"
-
-cat << EOF > "${python_file}"
-#!/usr/bin/env python3
-
-# Imports
-from samplesheet.samplesheet_check import run_sample_sheet_content_check
-from samplesheet.samplesheet_check import run_sample_sheet_check_with_metadata
-from utils.samplesheet import SampleSheet
-
-# Get auth header for portal
-auth_header = "Bearer ${PORTAL_TOKEN}"
-
-# Get samplesheet
-sample_sheet_path = "${SAMPLESHEET_FILE}"
-sample_sheet = SampleSheet(sample_sheet_path)
-
-# Check 1
-run_sample_sheet_content_check(sample_sheet)
-
-# Check 2
-async def set_and_check_metadata(sample_sheet, auth_header):
-    # Set metadata
-    loop = asyncio.get_running_loop()
-    error = await asyncio.gather(
-        sample_sheet.set_metadata_df_from_api(auth_header, loop),
-    )
-    
-    # run metadta check
-    run_sample_sheet_check_with_metadata(sample_sheet)
-    
-loop = asyncio.new_event_loop()
-set_and_check_metadata(sample_sheet, auth_header):
-loop.close()
-
-    
-
-
-EOF
-
-echo "Running samplesheet '${SAMPLESHEET_FILE}' through check script '${SAMPLESHEET_CHECK_SCRIPT}'" 1>&2
-conda run \
-  --name "${CONDA_ENV_NAME}" \
-  python3 "${python_file}"
-  
-echo "Test complete!" 1>&2
-rm "${python_file}"
-
-#############
-```
-
-## Testing through API
-
-This goes through running the samplesheet check against the API.
-
-This tests the deployed version of the samplesheet check, rather than the local file content.
-
-You can use the curl binary to make a POST request to the API.  
-
-This script expects the user to have the following environment variables:
-  * `PORTAL_TOKEN` (can be obtained from the data.umccr.org home page)
-
-```bash
-API_URL="https://api.sscheck.prod.umccr.org"  # Dev URL: https://api.sscheck.dev.umccr.org
-SAMPLESHEET_FILE="SampleSheet.csv"
-
-curl \
-  --location \
-  --request POST \
-  --header "Authorization: Bearer ${PORTAL_TOKEN}" \
-  --form "logLevel=ERROR" \
-  --form "file=@${SAMPLESHEET_FILE}" \
-  "${API_URL}"
-```
-
```

Comment:
This README test section still applicable? Or perhaps, you have other plan with it or rearranging/refactoring it? 🙄 

Especially this last section help me how to leverage sscheck API endpoint, you know. Perhaps about time create `/docs` etc.

If you have a plan, that's ok with this changes. We can always dig to git history, if any.

Reply:
Yeah could maybe put this to `docs` instead of removal. Well it is because I put in local invoke via `sam` cli, and so [testing locally could be done via the curl API](https://github.com/umccr/samplesheet-check-backend#testing-through-api) instead of this script. The local invoke API could be configured so that it will run locally but could use the remote metadata endpoint with just `make start` (and use the curl/postman command to check the local samplesheet file)

Reply:
Yup. Developer guide | User guide would be handy, Will.